README.md

```diff
@@ -1,2 +1,87 @@
-# antrea-operator
-Antrea Operator for Openshift
+# Antrea Operator
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Overview
+
+The Antrea Operator installs [Antrea](https://github.com/vmware-tanzu/antrea/)
+CNI plugin on an OpenShift Kubernetes cluster.
+The operator will be deployed in the early phases of cluster
+deployment, and it will take care of deploying Antrea components:
+
+* The antrea-controller deployment
+* The antrea-agent daemonset
+
+For Openshift 4 clusters, the antrea-operator monitors the network CR in the
+config.openshift.io namespace to update the container network CIDRs used by
+Antrea.
+
+The antrea-operator also monitors a dedicated CR, applies
+changes to antrea-controller and abtrea-agent configuration, and restart the
+relevant pods so that the relevant configuration changes are picked up.
+
+## Try it out
+
+Generate install-config.yaml by using openshift-install command.
+```
+$ openshift-install --dir=MY_CLUSTER create install-config
+```
+
+Edit `MY_CLUSTER/install-config.yaml` to update networking section.
+Change `networkType` to `antrea`(case insensitive).
+Set container network CIDRs `clusterNetwork` in `MY_CLUSTER/install-config.yaml`.
```

Comment:
antrea does not care about clusterNetwork, how about mentioning service network CIDR here as well